docs/create-packages/Select-assemblies-referenced-by-projects.md

```diff
@@ -9,25 +9,44 @@ ms.topic: conceptual
 
-> [!Note]
-> Explicit assembly references are related to .NET assemblies. It is not a method to distribute native assemblies that are P/Invoked by a managed assembly.
+## Recommendation: Package dependencies and include/exclude assets
 
-## `PackageReference` support
+Our recommendation is to have one package per assembly, in which case you can use `PrivateAssets="compile"` in your `PackageReference` or `ProjectReference`. This provides more flexibility and minimizes risk that two different packages will contain different versions of the same assembly, preventing NuGet from being able to manage versions.
 
-When a project uses a package with `PackageReference` and the package contains a `ref\<tfm>\` directory, NuGet will classify those assembles as compile-time assets, while the `lib\<tfm>\` assemblies are classified as runtime assets. Assemblies in `ref\<tfm>\` are not used at runtime. This means it is necessary for any assembly in `ref\<tfm>\` to have a matching assembly in either `lib\<tfm>\` or a relevant `runtime\` directory, otherwise runtime errors will likely occur. Since assemblies in `ref\<tfm>\` are not used at runtime, they may be [metadata-only assemblies](https://github.com/dotnet/roslyn/blob/main/docs/features/refout.md) to reduce package size.
+```xml
+<ItemGroup>
+  <ProjectReference Include="..\OtherProject\OtherProject.csproj" PrivateAssets="compile" />
+  <PackageReference Include="SomePackage" Version="1.2.3" PrivateAssets="compile" />
+</ItemGroup>
+```
 
-> [!Important]
-> If a package contains the nuspec `<references>` element (used by `packages.config`, see below) and does not contain assemblies in `ref\<tfm>\`, NuGet will advertise the assemblies listed in the nuspec `<references>` element as both the compile and runtime assets. This means there will be runtime exceptions when the referenced assemblies need to load any other assembly in the `lib\<tfm>\` directory.
+This gives your package consumers the flexibility to directly reference the package to get the compile time assets.
 
-> [!Note]
-> If the package contains a `runtime\` directory, NuGet may not use the assets in the `lib\` directory.
+## Not recommended: Multiple assemblies in one package
+
+`PackageReference` and `packages.config` have different features available. Whether you want to support your package consumers who use `PackageReference`, `packages.config`, or both, changes how you must author your package.
+
+NuGet's MSBuild Pack target does not support automatically including project references in the package. See [this issue](https://github.com/NuGet/Home/issues/3891), which lists some ways different developers have achieved their goal, or consider community developed tools such as [NuGetizer](https://github.com/devlooped/nugetizer). See the docs on [including content in a package](../reference/msbuild-targets.md#including-content-in-a-package) for how to use the `PackagePath` MSBuild item metadata to put files in any location in the package.
```

Comment:
Please don't link `this`. The text in absence of links should read as normal text. How about

> There is a [GitHub issue]((https://github.com/NuGet/Home/issues/3891)), which lists some ways...